.github/workflows/pkgci_regression_test.yml

```diff
@@ -234,7 +238,11 @@ jobs:
           - name: cpu_llvm_task
             models-config-file: models_cpu_llvm_task.json
```

Comment:
mi250:
```
============================== slowest durations ===============================
597.91s call     shark-test-suite-models/sdxl/test_unet.py::test_run_unet_cpu
292.26s call     shark-test-suite-models/sdxl/test_vae.py::test_run_vae_cpu
38.20s call     shark-test-suite-models/sdxl/test_unet.py::test_compile_unet_cpu
17.20s call     shark-test-suite-models/sdxl/test_clip.py::test_compile_clip_cpu
```
https://github.com/iree-org/iree/actions/runs/10498668335/job/29084321533#step:7:126

😬 

might be an improvement over the w7900 if we at least have enough CPU cores / runners to run jobs in parallel

Reply:
Yeah mi250 is a bit slower than w7900. Not sure how the mi300 fares, but it was initially why I picked w7900